files/install-requirements.yaml

```diff
@@ -34,6 +34,13 @@
     pip:
       name: git+https://github.com/packit-service/sandcastle
 
+  - name: Upgrade some packages to PyPI version
+    pip:
+      name:
+        - ogr
+        - PyGithub
```

Comment:
I don't like this, we can't easily affect pygithub version in fedora

Reply:
That's true, but for now we need PyGithub 1.43 and in Fedora is only 1.39.
Someone has to package into Fedora missing dependency needed by PyGithub.
I would prefer to add there a link to Bugzilla (https://bugzilla.redhat.com/show_bug.cgi?id=1595244) so we can later on change it back to Fedora RPM package.

Reply:
@TomasTomecek do you have some better solution?

Reply:
Dependency for PyGithub `python-deprecated` is already packaged https://bugzilla.redhat.com/show_bug.cgi?id=1595244#c16
Then we need to try if PyGithub can be built into Fedora.